include/cantera/transport/MixTransport.h

```diff
@@ -149,15 +149,9 @@ class MixTransport : public GasTransport
                                   size_t ldx, const doublereal* const grad_X,
                                   size_t ldf, doublereal* const fluxes);
 
-    virtual void init(thermo_t* thermo, int mode=0, int log_level=0);
+    virtual void init(ThermoPhase* thermo, int mode=0, int log_level=0);
 
 protected:
-    //! Calculate the pressure from the ideal gas law
-    doublereal pressure_ig() const {
-        return (m_thermo->molarDensity() * GasConstant *
-                m_thermo->temperature());
-    }
-
```

Comment:
Just double checking that removing this entirely is the intention, since you created a new function in `MultiTransport.h`.

Reply:
Yes, in the `MixTransport` class, it was a `protected` member of that was not actually used.

In the case of `MultiTransport`, the method is used, so I moved the definition to the `.cpp` file because doing so eliminates the need to have the full definition of `ThermoPhase` available in `MultiTransport.h`.